services/services.go

```diff
@@ -491,6 +494,31 @@ func indexPageDataUpdater(wg *sync.WaitGroup) {
 	}
 }
 
+func ethStoreStatisticsDataUpdater(wg *sync.WaitGroup) {
+	firstRun := true
+	for {
+		data, err := getEthStoreStatisticsData()
+		if err != nil {
+			logger.Errorf("error retrieving ETH.STORE statistics data: %v", err)
+			time.Sleep(time.Second * 10)
+			continue
+		}
+
+		cacheKey := fmt.Sprintf("%d:frontend:ethStoreStatistics", utils.Config.Chain.Config.DepositChainID)
+		err = cache.TieredCache.Set(cacheKey, data, time.Hour*24)
+		if err != nil {
+			logger.Errorf("error caching indexPageData: %v", err)
```

Comment:
Copy&Paste error ("indexPageData" versus "ethStoreStatistics").

Reply:
Fixed in new commit